WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -238,9 +244,9 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required]InputsReque
 						}
 
 						// Check if immature.
-						if (getTxOutResponse.Confirmations <= 100)
+						if (getTxOutResponse.IsCoinBase)
 						{
-							if (getTxOutResponse.IsCoinBase)
+							if (getTxOutResponse.Confirmations <= 100)
```

Comment:
This change is somewhat unnecessary. Anyway if you are going to beautify it, then unnest it: `if (getTxOutResponse.IsCoinBase && getTxOutResponse.Confirmations <= 100)`

Reply:
I think some nesting is fine, because un-nesting can make the code complex.
For example nesting in https://github.com/zkSNACKs/WalletWasabi/pull/3637/commits/8cb32a2030ff51fe0a8d91457a560a2fa5bc055d made CodeScene pass.

Reply:
Looks better for me without nesting.

Reply:
done.